helpers/lib/cdnify.js

```diff
@@ -56,7 +56,14 @@ module.exports = globals => {
             }
 
             if (protocol === 'webdav:') {
-                return [cdnUrl, 'content', path].join('/');
+                const supportedImageExtensions = ['jpg', 'jpeg', 'gif', 'png'];
+                const unsupportedPathSyntaxes = ['../', './'];
+                const searchParamIgnoredPath = path.split('?')[0];
```

Comment:
we should cover cases when pathname contains searchParams like 
`
'webdav:/img/brands-logos/main.png?cache_bust=1749005114992'`
and exclude them from filtering